### ISSUE TITLE
[Questtion] Missing netlify deploy step?

### ISSUE THREAD

**Issue by noobling** [redwoodjs/example-blog#50]
Hey I spent a fair amount of time trying to figure out why I was hitting a CORS error when pasting the netlify URL. Turns out you have to deploy the netlify app first e.g. `netlify deploy --prod` Not sure if this is worth adding or just me.

**Comment by cannikin** [comment]
Good point, we should add a note about that. We'd accept a PR if you want to take a stab at adding it? :)
